raiden/routing.py

```diff
@@ -208,7 +209,13 @@ def get_best_routes_internal(
         # The complete route includes the initiator, add it to the beginning
         complete_route = [Address(from_address)] + neighbour.route
 
-        available_routes.append(RouteState(complete_route, neighbour.channelid))
+        available_routes.append(
+            RouteState(
+                route=complete_route,
+                forward_channel_id=neighbour.channelid,
+                estimated_fee=FeeAmount(0),
```

Comment:
The fee here is 0, since this is the internal routing, right? Can you add a comment explaining this?